src/context/like-video-context.jsx

```diff
@@ -0,0 +1,68 @@
+import axios from "axios";
+import { createContext, useState } from "react";
+
+const LikeVideoContext = createContext();
+
+const LikeVideoProvider = ({ children }) => {
+  const [likeVideos, setLikeVideos] = useState([]);
+
+  const addToLikedVideoHandler = (video) => {
+      
+  const encodedToken = localStorage.getItem("token");
+    (async () => {
+      try {
+        const response = await axios.post(
+          "/api/user/likes",
+          {
+            video,
+          },
+          {
+            headers: {
+              authorization: encodedToken,
+            },
+          }
+        );
+        if (response.status === 201) {
+          setLikeVideos(response.data.likes);
+        }
+      } catch (err) {
+        console.error(err);
+      }
+    })();
+  };
+
+  const removeFromLikeVideoHandler = (video) => {
+      
+  const encodedToken = localStorage.getItem("token");
+    (async () => {
+      try {
+        const response = await axios.delete(`/api/user/likes/${video._id}`, {
+          headers: {
+            authorization: encodedToken,
+          },
+        });
+        console.log(response);
```

Comment:
`console.log` can be removed if not needed.

Reply:
sure